test/pytorch_test_base.py

```diff
@@ -286,6 +286,9 @@
     'TestTorchDeviceTypeXLA': {
         'test_float_to_int_conversion_finite_xla',  # different behavior than numpy when casting float_max/min to int types
     },
+    'TestTensorDeviceOpsXLA': {
+        'test_svd_square_xla',  # FIXME: wrong result
```

Comment:
Do we have an issue (nostale) open for this?

Reply:
Hmm not yet, tbh every FIXME or lines without FIXME in this file need to be investigated lol